stable/kubecost/values.yaml

```diff
@@ -57,6 +57,9 @@ cost-analyzer:
     enabled: true
     # storageClass: "-"
 
+  podSecurityPolicy:
+    enabled: false
```

Comment:
Do we want that ? Was it the default behavior before ?

Reply:
The default for kubecost users is true :/. 

Should we disable it for the main mgmt or better use kubefed overrides to disable it in the federated Addon kubecost when placed in the mgmt cluster (which we might have a label for 🤔 ) ?

Reply:
It was defaulted to false in the past, the latest releases changed the default to true which caused this to break in kommander. I think maybe the correct fix is to use a unique name for PSP so there won't be name conflicts, I can try and put up a pr against kubecost

Reply:
okay, sounds good. Let's track it in a jira ticket. It sounds good for me disabling now.

Reply:
https://jira.d2iq.com/browse/D2IQ-74182 will open a pr upstream

Reply:
Great @gracedo :).

Reply:
https://github.com/kubecost/cost-analyzer-helm-chart/pull/773